code/modules/crew_objectives/medical_objectives.dm

```diff
@@ -86,10 +86,6 @@
 			units_total += container.reagents?.get_reagent_amount(target_chemical)
 	return units_total >= target_amount
 
-/datum/objective/crew/noinfections
-	explanation_text = "Make sure there are no living crew members with harmful diseases at the end of the shift."
-	jobs = JOB_NAME_VIROLOGIST
-
 /datum/objective/crew/noinfections/check_completion()
```

Comment:
this needs removal too otherwise it will still exist

Reply:
Cleaned it up.